src/constraints/outage_constraints.jl

```diff
@@ -116,7 +116,7 @@ function add_MG_production_constraints(m,p)
 	# Electrical production sent to storage or export must be less than technology's rated production
 	@constraint(m, [t in p.techs.elec, s in p.s.electric_utility.scenarios, tz in p.s.electric_utility.outage_start_time_steps, ts in p.s.electric_utility.outage_time_steps],
 		m[:dvMGProductionToStorage][t, s, tz, ts] + m[:dvMGCurtail][t, s, tz, ts] <=
-		(p.production_factor[t, tz+ts-1] + p.unavailability[t][tz+ts-1]) * p.levelization_factor[t] * m[:dvMGRatedProduction][t, s, tz, ts]
+		(p.production_factor[t, time_step_wrap_around(tz+ts-1, time_steps_per_hour=p.s.settings.time_steps_per_hour)] + p.unavailability[t][time_step_wrap_around(tz+ts-1, time_steps_per_hour=p.s.settings.time_steps_per_hour)]) * p.levelization_factor[t] * m[:dvMGRatedProduction][t, s, tz, ts]
     )
```

Comment:
So the wraparound function is required for data from the initial problem (e.g., production factors and site load) and NOT required for the microgrid state of charge because that's only using the starting time period SOC from the full-year problem and then using the (tz,ts) tuple instead of an "absolute-value" time step that could otherwise exceed 8760.  Is that right?

I spot-checked a case and the m[:dvStorageEnergy"] reconciles correctly for the long-duration case in the testset - just making sure I understand.

Reply:
If I'm understanding your question correctly, yes. The outages specific decision variables have separate indices for tz and ts so do not need the wrap around. For these DVs, the ts index is outage_time_steps which starts at 1. A time step can only go beyond the end of the year when tz and ts are summed, as they are when accessing full year data/DVs.

Reply:
> If I'm understanding your question correctly, yes. The outages specific decision variables have separate indices for tz and ts so do not need the wrap around. For these DVs, the ts index is outage_time_steps which starts at 1. A time step can only go beyond the end of the year when tz and ts are summed, as they are when accessing full year data/DVs.

Yup, that matches my read of what's happening, and why you only need that wraparound function for when you sum relevant parameters/dv's on subsets of p.time_steps that cover periods 1 and 8760 (but not the full set).  Thanks for confirming!